### PR TITLE
NO-ISSUE: Update images in project-setup

### PR DESCRIPTION
**Pull request by kbowers-ibm**
Updating the images in project-setup to use images using the new dmn editor

### CODE REVIEW

**Review by justlmills — APPROVED** [review]

File: getting-started/project-setup.adoc [hunk 1]
```diff
@@ -130,7 +128,7 @@ image::automatic-layout.png[Automatic Layout]
 +
 image::dmn-first-view.png[DMN First view]
 
-. If you click *Age* and then click the *Properties* icon on the right, you will open a pane for the input.
+. If you click the *Age* input node and then click the *Properties* icon on the right, you will open a pane for the input.
```
Comment (jomarko) [inline getting-started/project-setup.adoc:131]: I think officially DMN standard does not have the term input node, just Input Data node.

**Review by jomarko — COMMENTED** [review]
Thank you for updates @kbowers-ibm 

maybe we want to highlight also the edit button in the `open-decision.png`?

One more question, what is your opinion about using unified color for highlighting in screenshots?

In this PR I stored the HEX code, not sure why it is not on main anymore https://github.com/IBM/bamoe-docs/pull/103/files

**Comment by kbowers-ibm** [comment]
@jomarko Thanks for the review!
`maybe we want to highlight also the edit button in the open-decision.png?`

I tried, but it looked messier with the highlight in my opinion. I think for the classic dmn editor it was needed, but now that the edit button is so prominent, it isn't required. Happy to change if you disagree though!

> One more question, what is your opinion about using unified color for highlighting in screenshots?

I'd be up for that later down the road, but just wanted to replace red with red for now, and it also matches the other screenshots further down the page.

**Review by LightGuard — APPROVED** [review]

**Comment by LightGuard** [comment]
Any reason *not* to merge?

**Comment by kbowers-ibm** [comment]
@LightGuard I believe it's good to go 👍

**Pull request merged**